README.md

```diff
@@ -16,11 +16,13 @@ yarn add freedom-accounts-util
 ```
 const accounts = require('freedom-accounts-util');
 
-accounts.configure({base_url: '', path: '', client_id: '', client_secret: '', client_expiry: 300, server_expiry: 300});
+accounts.configure({base_url: '', path: '', client_id: '', client_secret: '', client_expiry: 300, server_expiry: 300, disable_caching: false});
```

Comment:
Should this be a config option? Wouldn't this be more useful as a function/toggle that the program can call to temp enable/disable cache instead of configuring as cache disabled?

Reply:
I think this should be okay since `accounts.configure` already merges into the configuration so it can be
```
accounts.configure({disable_caching: true});
// do something without caching
accounts.configure({disable_caching: false});
// caching is back here
```

Adding functions to set this would just be shortcuts to the above code.